packages/modules/viewer-react/src/tests/components/iModel/IModelLoader.test.tsx

```diff
@@ -147,6 +155,15 @@ describe("IModelLoader", () => {
     jest.spyOn(UiItemsManager, "register");
     jest.spyOn(UiItemsManager, "unregister");
 
+    act(() =>
```

Comment:
Due to the change in react 18 to rendering, need to find a way to rewrite the tests in this file since render no longer returns a RenderResult